app/models/gsa18f/procurement.rb

```diff
@@ -26,34 +26,35 @@ def self.purchase_amount_column_name
     validates :product_name_and_description, presence: true
     validates :recurring_interval, presence: true, if: :recurring
 
-    def add_steps
-      steps = [
-        Steps::Approval.new(user: User.for_email(Gsa18f::Procurement.approver_email)),
-        Steps::Purchase.new(user: User.for_email(Gsa18f::Procurement.purchaser_email)),
-      ]
-      proposal.add_initial_steps(steps)
-    end
-
     def self.relevant_fields(recurring)
-      fields = [:office, :justification, :link_to_product, :quantity,
-        :date_requested, :urgency, :additional_info, :cost_per_unit,
-        :product_name_and_description, :recurring]
+      fields = self.default_fields
+
       if recurring
         fields += [:recurring_interval, :recurring_length]
       end
+
       fields
     end
 
-    def relevant_fields
-      Gsa18f::Procurement.relevant_fields(recurring)
+    def self.default_fields
```

Comment:
I would really like to clean this up further with a Form object, but found myself going down a rabbit hole so decided to move onto other things for the time being